modules/candidate_parameters/ajax/formHandler.php

```diff
@@ -425,11 +425,11 @@ function editConsentStatusFields($db, $user)
 
         // Process posted data
         // Empty strings and type null are not passed (null is passed as a string)
-        $status     = ($_POST[$consentName] !== 'null') ?
+        $status     = (isset($_POST[$consentName]) && $_POST[$consentName] !== 'null') ?
```

Comment:
You can use null coalesce on these lines here to make them simpler, e.g.

```php
$data =  $_POST[$consentName . '_date'] ?? null;
```

Reply:
nope because the second condition is checking to see if the variable is a string 'null', not null

Reply:
Thanks, I missed that.